packages/graph/hash_graph/lib/graph/src/store/migration.rs

```diff
@@ -0,0 +1,65 @@
+use async_trait::async_trait;
+use error_stack::Result;
+
+use super::error::MigrationError;
+
+#[derive(Debug, Default, PartialEq, Eq)]
+pub enum MigrationState {
+    Applied {
+        applied_at_utc: i64,
```

Comment:
Maybe we could use `DateTime` here instead? Or even `Timestamp<Migration>`?

Reply:
I have a feeling `TimeStamp<Migration>` may be a little overkill? I will perhaps use DateTime isntead